sdv/multi_table/hma.py

```diff
@@ -462,6 +481,9 @@ def _model_tables(self, augmented_data):
             )
 
             if not table.empty:
+                self._set_extended_columns_distributions(
+                    self._table_synthesizers[table_name], table_name, table.columns
+                )
```

Comment:
why do you need this both here and in `_get_extension`?

Reply:
- There is 2 synthesizers that are created. One at the table level (in `_model_tables`) and one per each FK grouping (the grouped child rows in `_get_extension`). 
- We need to call it twice to make sure that in both situations, the extension columns use `truncnorm`. 
- If I called it at just the table level, the synthesizer for the grouped child rows will just be a default GaussianCopula (all distributions set to beta). This would result in the constant values with 0/1 child rows (current behavior).